### DISCUSSION TITLE
最近使用search2ai为GPT联网出现错误

### DISCUSSION THREAD

**Discussion by SpeedupMaster** [BurnHair/docs#103] [技术支持]
6月2日通过search2ai为GPT联网可以正常使用，但是昨天发现需要联网时就会报错。
```
{
  "error": {
    "message": "Invalid value for 'content': expected a string, got null. (request id: 20240609113056447889482LhlQhS3K)",
    "type": "invalid_request_error",
    "param": "messages.[7].content",
    "code": null
  }
}
```
使用官方的API就没问题。对于不需要联网的问题回复也没有问题，只有当要联网的查询的时候就会报错。
<img width="1386" alt="Snipaste_2024-06-09_19-23-53" src="https://github.com/BurnHair/docs/assets/130642629/e670a4bf-a0c8-4869-9156-2c0d35df08de">
<img width="1339" alt="Snipaste_2024-06-09_19-24-29" src="https://github.com/BurnHair/docs/assets/130642629/4cccba8e-1269-470f-a2f5-602adbee2038">
<img width="671" alt="Snipaste_2024-06-09_19-28-39" src="https://github.com/BurnHair/docs/assets/130642629/38e03543-89eb-4af8-996d-4ef58553e438">
<img width="672" alt="Snipaste_2024-06-09_19-28-52" src="https://github.com/BurnHair/docs/assets/130642629/40e90ca8-9992-48dd-953d-82a9ae7bc534">

**Comment by BennyThink** [comment]
可能是因为   content filter。我看看吧

**Reply by SpeedupMaster** [reply]
好的

**Reply by BennyThink** [reply]
应该修好了，你再试试看
<img width="896" alt="image" src="https://github.com/BurnHair/docs/assets/14024832/c544f2c8-c675-4f0a-823a-3819f94bf12a">

**Reply by SpeedupMaster** [reply]
可以了
